tests/css_yaml/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -31,6 +31,9 @@ a {
 }
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 ---
+
+
+
```

Comment:
This looks like a regression to me.

Reply:
We'll print the front(-)matter as-is from now on